rhel/SPECS/pmm-managed.spec

```diff
@@ -56,14 +56,19 @@ make release
 install -d -p %{buildroot}%{_bindir}
 install -d -p %{buildroot}%{_sbindir}
 install -p -m 0755 bin/pmm-managed %{buildroot}%{_sbindir}/pmm-managed
+install -p -m 0755 bin/pmm-managed %{buildroot}%{_sbindir}/pmm-managed-init
 
 
 %files
 %license src/%{provider_prefix}/LICENSE
 %doc src/%{provider_prefix}/README.md
 %{_sbindir}/pmm-managed
+%{_sbindir}/pmm-managed-init
 
 
 %changelog
+* Tue Fed 11 2020 Mykyta Solomko <mykyta.solomko@percona.com> - 2.0.0-14
```

Comment:
Please fix it 'Feb'

Reply:
from log:
```
error: bad date in %changelog: Tue Fed 11 2020 Mykyta Solomko <mykyta.solomko@percona.com> - 2.0.0-14

error: query of specfile SOURCES/pmm-managed.spec failed, can't parse
```

Reply:
Mine apologize, that I have fixed this typo - just wondering whether it all will together work :)